src/datasets/tasks/question_answering.py

```diff
@@ -0,0 +1,44 @@
+from dataclasses import dataclass
+from typing import Dict
+
+from ..features import Features, Sequence, Value
+from .base import TaskTemplate
+
+
+@dataclass
+class QuestionAnswering(TaskTemplate):
+    task = "question-answering"
+    input_schema = Features({"question": Value("string"), "context": Value("string")})
+    label_schema = Features(
+        {
+            "answers": Sequence(
+                {
+                    "text": Value("string"),
+                    "answer_start": Value("int32"),
+                }
+            )
+        }
+    )
+    question_column: str
+    context_column: str
+    answers_column: str
+
+    def __init__(self, question_column="question", context_column="context", answers_column="answers"):
```

Comment:
any reason to do have the default values in `__init__` instead this ?
```python
    question_column: str = "question"
    context_column: str = "context"
    answers_column: str = "answers"
```

Reply:
good catch! this is now fixed with the new `__post_init__` usage